Apromore-Clients/manager-client/src/main/java/org/apromore/manager/client/ManagerServiceImpl.java

```diff
@@ -110,10 +113,50 @@
 
     private boolean isGEDMatrixReady = true;
     
-    private static final Logger LOGGER = LoggerFactory.getLogger(ManagerServiceImpl.class);
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
+// Implementation of ManagerService
 
-    // Implementation of ManagerService
+    /**
+     * Logout, in Keycloak, user sessions associated with said user.
+     *
+     * @param username The username of the user to logout.
+     *
+     * @return <code>true</code> if the user was logged-out, <code>false</code> otherwise.
+     */
+    @Override
+    public boolean logoutUserAllSessions(final String username) throws Exception {
+        Boolean restRespResult = false;
+
+        final RestTemplate restTemplate = new RestTemplate();
```

Comment:
Check whether REST template needs to have timeout configured.  See https://stackoverflow.com/questions/11537591/resttemplate-default-timeout-value

Reply:
Added RestTemplate bean with connection/read timeout, per review comment [DONE].